src/components/elements/annotationSidePanel/components/annotationWizard/steps/AnnotationSummaryStep.tsx

```diff
@@ -33,15 +35,37 @@ type Props = {
  * @param superClass The selected super class
  * @param schemaTitle The title of the super class schema
  * @param formValues The current form values of the annotation form
+ * @param SetFieldValue Function to set the value of a field in the annotation wizard form
  * @returns JSX Component
  */
 const AnnotationSummaryStep = (props: Props) => {
-    const { superClass, schemaTitle, formValues } = props;
+    const { superClass, schemaTitle, formValues, SetFieldValue } = props;
+
+    /* Hooks */
+    const trigger = useTrigger();
 
     /* Base variables */
     const annotationTarget = useAppSelector(getAnnotationTarget);
+    const tourAnnotationWizardFormValues = useAppSelector(getAnnotationWizardFormValues);
     let motivationDescription: string;
 
+    /* Trigger for tour annotation wizard form values */
+    trigger.SetTrigger(() => {
```

Comment:
using ` const { class: classValue, jsonPath, annotationValues } = tourAnnotationWizardFormValues;` then accessing the given values might be a bit cleaner

Reply:
Also, can you reuse this function elsewhere? seems like it's a bit repetitive

Reply:
Can only do it for jsonPath and annotationValues, class cannot be a variable as it is a keyword for identifying a class

Reply:
Moved the part I could move of the trigger to a utility function, which does not really comply with what a utility function is, but prevents the repeat of the code